plugins/modules/ec2_vpc_route_table_info.py

```diff
@@ -260,7 +260,7 @@ def list_ec2_vpc_route_tables(connection, module):
 
 def main():
     argument_spec = dict(
-        filters=dict(default=None, type="dict"),
+        filters=dict(default={}, type="dict"),
     )
```

Comment:
Can you also please add `default: {}` in the documentation https://github.com/ansible-collections/amazon.aws/pull/1670/files#diff-4438a05c875c0db8b97afe9ea0cfdcd3e52c6ad2925061e66464d68a293ca07dR22

Reply:
I've updated the documentation with the default